datameta/api/openapi.yaml

```diff
@@ -736,6 +768,15 @@ components:
       items:
         $ref: "#/components/schemas/SubmissionResponse"
 
+    GroupNameChange:
```

Comment:
Let's stick to one naming scheme, maybe `GroupUpdateRequest` in accordance with `FileUpdateRequest`

Reply:
Done